kernel/reduction.mli

```diff
@@ -29,7 +37,11 @@ val default_cfg : red_cfg
 *)
 
 val reduction : red_cfg -> Signature.t -> term -> term
-(** [reduction sg red te] reduces the term [te] following the strategy [red]
+(** [reduction cfg sg te] reduces the term [te] following the configuration [cfg]
+    and using the signature [sg]. *)
+
+val quick_reduction : red_target -> Signature.t -> term -> term
```

Comment:
It's not exactly the same, this version is slightly more efficient (hence the "quick" keyword which should actually have rather been "fast") because it relies on a stripped down version of state_whnf/snf that doesn't remember current rewriting position and doesn't try to log anything.